### DISCUSSION TITLE
Reduced Hoods each week

### DISCUSSION THREAD

**Discussion by Ignis231** [THGSCST/Vices-and-Sins-MOD#9] [Q&A]
Hi,
I know there has been rebalancing as part of this mod, and high quality hoods are rarer, in fact high quality hoods are only about 20% of what generates.
However, I'm finding far fewer actual hoods are being generated as well. Not sure if this is intended, or is a by product of lower intelligence hoods recruiting, but it seems a bit off balance. I'm at week 15, one gang has already been wiped out, and most weeks I'm only getting about 4 hoods in the talent pool. Is this intended?
Do you know what value in which file is what controlling the amount of hoods generated?

**Comment by THGSCST** [comment]
Greetings,
Regarding quality and quantity in recruitment, this mod changed two key points:
- High quality hoods only have a `weight` of 1 (this makes them very difficult to appear).
- The frequency of gym and pool hall type businesses has been increased (which makes recruiting easier)

With these changes, the best way to recruit is to visit as many gyms and pool halls as possible, doing this almost every week, and as you already know, a recruiter with high intelligence obtains better results. The files I modified to get these changes were: `Economics.txt` (parameters: _City Capacity, City Cap Min, City Cap Max_ and _Numbers Present_) for business frequency and `Character Generation.txt` (parameter _Weighting_ or _Wgt_) for the frequency of hoods generation.

Regarding the gang being defeated in week 15, it is difficult to diagnose the causes. Do you have a saved game? It usually happens that a gang that has lost many fights has difficulty replacing hoods, because now its recruiting team are smaller too. There is no defeat for having zero hoods, if a gang reaches zero hoods, the game will randomly generate one for the next week. However, a single hood is not capable of defending the office, the other gangs are active in sending hostile visits to rival offices. The other cause of defeat is bankruptcy.

**Reply by Ignis231** [reply]
No the being defeated by week 15 is not an issue, I just meant the game is progressing, things are happening, but I'm hiring 2* people to run businesses, I've not had anyone with 4* intelligence appear at all. Either the total hoods generated or the weighting needs a bit of rebalance (IMHO).

**Reply by Ignis231** [reply]
Tring to adjust the numbers but getting crashes, so must be getting something wrong. Is there a max difference between the cmin and vmax, beyond the total ( Int Cmin 0 , VMax 192  , would this work?)
Also the weight, any balancing that needs to exist, or should any number between 1-9 should work for any of them?

**Reply by THGSCST** [reply]
Yes, it will work because the sum of CMax+VMax is less than 256 and VMax is not zero. Your error looks like a problem with the TSV (tab-separated-values) formatting. Note that each value is accompanied by a tab character, there is no space:
```
# Wgt	Int		Ars		Fst		Drv		Biz		Stl		Xpl		Itm		Fir		Knv		Org
# Wgt	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax
5	0	192	16	48	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	# 1. Firebug
```

Regarding the frequency of the hoods, I will do a second check soon. For now you can change the Weighting (Wgt) parameter from 2 to 3 and from 5 to 7, like this:
```
# Wgt	Int		Ars		Fst		Drv		Biz		Stl		Xpl		Itm		Fir		Knv		Org
# Wgt	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax	CMin	Vmax
7	0	128	16	48	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	# 1. Firebug
7	0	128	0	32	16	48	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	# 2. Brute
7	0	128	0	32	0	32	16	48	0	32	0	32	0	32	0	32	0	32	0	32	0	32	# 3. Tailgater
7	0	128	0	32	0	32	0	32	16	48	0	32	0	32	0	32	0	32	0	32	0	32	# 4. Salesman
7	0	128	0	32	0	32	0	32	0	32	16	48	0	32	0	32	0	32	0	32	0	32	# 5. Wallflower
7	0	128	0	32	0	32	0	32	0	32	0	32	16	48	0	32	0	32	0	32	0	32	# 6. Watchmaker [Starting Hood (n=10)]
7	0	128	0	32	0	32	0	32	0	32	0	32	0	32	16	48	0	32	0	32	0	32	# 7. Bully [Starting Hood]
7	0	128	0	32	0	32	0	32	0	32	0	32	0	32	0	32	16	48	0	32	0	32	# 8. Triggerman [Starting Hood (n>=8)]
7	0	128	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	16	48	0	32	# 9. Butcher
7	0	128	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	0	32	16	48	# 10. Planner [Starting Hood (n>=7)]
3	64	128	32	32	16	24	16	24	16	24	16	24	32	32	16	24	16	24	16	24	16	24	# 11. Anarchist [Starting Hood]
3	64	128	16	24	32	32	16	24	16	24	16	24	16	24	32	32	16	24	32	32	16	24	# 12. Ruffian
3	64	128	16	24	16	24	32	32	32	32	16	24	16	24	16	24	16	24	16	24	32	32	# 13. Manager
3	64	128	16	24	16	24	16	24	16	24	32	32	16	24	16	24	16	24	16	24	32	32	# 14. Coordinator 
3	64	128	16	24	16	24	16	24	16	24	16	24	16	24	16	24	32	32	16	24	16	24	# 15. Deadeye
1	128	128	48	16	24	16	24	16	24	16	24	16	48	16	24	16	24	16	24	16	24	16	# 16. Pyromaniac
1	128	128	24	16	48	16	24	16	24	16	24	16	24	16	48	16	24	16	48	16	24	16	# 17. Badass [Starting Hood (n>=9)]
1	128	128	24	16	24	16	24	16	48	16	24	16	24	16	24	16	24	16	24	16	48	16	# 18. Tycoon
1	128	128	24	16	24	16	48	16	24	16	24	16	24	16	24	16	24	16	24	16	48	16	# 19. Taskmaster 
1	128	128	24	16	24	16	24	16	24	16	48	16	24	16	24	16	48	16	24	16	24	16	# 20. Sharpshooter
### End of File ###
```

To facilitate editing, you can use Visual Studio Code to modify and Excel to view the result in table format. If you want to do some quick tests, you can edit hood number 11 "anarchist", it will always start with your gang, so it's easy to see the changes right at the beginning of the game.

**I forgot to mention something very important, which must be the cause: the way to find the best hoods is through the Diplomacy menu, in Snitch and Information on Unattached Gangsters. They are recruited using the Clipboard instead of the Map.**
